ontotext-yasgui-web-component/src/services/utils/visualisation-utils.ts

```diff
@@ -15,8 +15,9 @@ export class VisualisationUtils {
    * actually rendered before trying out to find it.
    * @param mode
    * @param orientation
+   * @param editorHeight if the mode is YASGUI, this height will be set for the yasqe. Default is 300px.
    */
-  static setYasqeFullHeight(mode: RenderingMode, orientation: Orientation): void {
+  static setYasqeFullHeight(mode: RenderingMode, orientation: Orientation, editorHeight = 300): void {
```

Comment:
Can we avoid having this fixed `300` px height hardcoded in more than one place? Now we have this default here, and also in the ontotext-yasgui the getEditorHeight method

Reply:
Yes, removed from here.